pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiTest.java

```diff
@@ -1219,7 +1219,9 @@ public void testDeleteTenantForcefully() throws Exception {
 
         // delete tenant forcefully
         admin.tenants().deleteTenant(tenant, true);
-        assertFalse(admin.tenants().getTenants().contains(tenant));
```

Comment:
fixing the problem this way is somehow closing the eyes in front of a problem.

if I delete a tenant I would like that if I run "pulsar-admin tenants list" the tenant is no more there

the "deleteTenant" operation is blocking and it must block until the operation succeeds and the system has a consistent view of the status of a such a "big entity".